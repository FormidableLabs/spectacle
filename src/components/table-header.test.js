import React from "react";
import { mount } from "enzyme";
import { mountToJson } from "enzyme-to-json";
import TableHeader from "./table-header";

describe("<TableHeader />", () => {
  test("should render correctly", () => {
    const wrapper = mount(
      <TableHeader>
        <tr><th>Table Content</th></tr>
      </TableHeader>
    );
    expect(mountToJson(wrapper)).toMatchSnapshot();
  });
});
