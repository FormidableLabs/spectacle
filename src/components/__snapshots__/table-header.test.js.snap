// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TableHeader /> should render correctly 1`] = `
<TableHeader>
  <thead>
    <tr>
      <th>
        Table Content
      </th>
    </tr>
  </thead>
</TableHeader>
`;
